.github/workflows/cpack.yml

```diff
@@ -18,18 +26,18 @@ jobs:
           - name: ubuntu
             version: "22.04"
             type: deb
-          - name: ubuntu
-            version: "20.04"
-            type: deb
+          # - name: ubuntu
```

Comment:
Hehe, I thought about that when I was re-reading it today. However, these lines are intentionally retained because commenting these lines is how those builds are in/excluded.